### PR TITLE
[RW-3334][risk=low] Publish CDRs through an intermediate project

### PR DESCRIPTION
**Pull request by calbach**
Blocked on: https://precisionmedicineinitiative.atlassian.net/browse/PD-4754

### CODE REVIEW

File: api/db-cdr/generate-cdr/libproject/devstart.rb [hunk 1]
```diff
@@ -90,9 +95,17 @@ def publish_cdr(cmd_name, args)
     # If you receive a prompt from "bq" for selecting a default project ID, just
     # hit enter.
     # TODO: Figure out how to prepopulate this value or disable interactivity.
-    common.run_inline %W{bq mk -f --dataset #{dest_dataset}}
+
+    # Copy through an intermediate project and delete after, per
+    # https://docs.google.com/document/d/1EHw5nisXspJjA9yeZput3W4-vSIcuLBU5dPizTnk1i0/edit
+    common.run_inline %W{bq mk -f --dataset #{ingest_dataset}}
     common.run_inline %W{./copy-bq-dataset.sh #{source_dataset} #{dest_dataset} #{env.fetch(:source_cdr_project)}}
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/libproject/devstart.rb]: I may be confused, but should this be like so?
```suggestion
    common.run_inline %W{./copy-bq-dataset.sh #{source_dataset} #{ingest_dataset} #{env.fetch(:source_cdr_project)}}
```

File: api/db-cdr/generate-cdr/libproject/devstart.rb [hunk 2]
```diff
@@ -90,9 +95,17 @@ def publish_cdr(cmd_name, args)
...
 
+    common.run_inline %W{bq mk -f --dataset #{dest_dataset}}
+    common.run_inline %W{./copy-bq-dataset.sh #{source_dataset} #{dest_dataset} #{env.fetch(:ingest_cdr_project)}}
```
Comment (freemabd) [inline api/db-cdr/generate-cdr/libproject/devstart.rb]: And this 
```suggestion
    common.run_inline %W{./copy-bq-dataset.sh #{ingest_dataset} #{dest_dataset} #{env.fetch(:ingest_cdr_project)}}
```

**Comment by calbach** [comment]
Had to switch to a TTL instead of an immediate delete. BQ dataset is "still in use" for a while after the copy is performed.

**Comment by calbach** [comment]
Tested on synthetic data, passed `--table_prefixes cb_,ds_`

**Pull request merged**